### DISCUSSION TITLE
Billing library 3 support for RN 0.59?

### DISCUSSION THREAD

**Discussion by ksendra** [hyochan/react-native-iap#1565] [Q&A]
Hi, I have an older application running on RN 0.59 and react-native-iap 2.5.5. npm install of react-native-iap does not bump the version for me.

Is there anything I can do on RN 0.59 to make my application support Billing Library 3 on Android? 

Thanks!

**Comment by gamenic-arim** [comment]
@ksendra have you found the solution?
